reliquery/storage.py

```diff
@@ -308,7 +302,7 @@ def put_metadata(self, path: Storage, metadata: Dict):
         )
 
     def get_metadata(self, path: StoragePath, root_key: str) -> Dict:
-        dirs = ["arrays", "html", "text", "images", "json", "pandasdf"]
+        dirs = ["arrays", "html", "text", "images", "json", "pandasdf", "file"]
```

Comment:
Are we using "file" or "files" I see it used differently. This is a good example of why we should have this list saved in a central location. That can be saved for another pull request, but I would figure out which version to use here.